src/modules/land_detector/RoverLandDetector.h

```diff
@@ -56,7 +57,11 @@ class RoverLandDetector : public LandDetector
 	bool _get_ground_contact_state() override;
 	bool _get_landed_state() override;
 
+
 private:
+	// Program crashes when Subscriptor declared here
+	//uORB::Subscription _vehicle_status_sub{ORB_ID(vehicle_status)};
```

Comment:
Can you revert this change and we'll work through whatever issue you hit?

Reply:
Hello dagar, apologies for delay, was away for Xmas.
Have just reverted the changes on my branch.
Please let me know if there is anything else I can help with